pdf_converter.py

```diff
@@ -106,13 +126,20 @@ def main(vn_only=True):
         ""
     )
 
+    # Centering images in html_file by replace <p> with <p align="center"> for lines that have
+    # <img> tag
+
+    for line in filedata.splitlines():
+        if "<img " in line:
+            new_line = line.replace("<p>","<p align=\"center\">")
```

Comment:
Okie, để cho chắc thì em thêm một cái assertion nữa để chắc chắc việc em "để ý" là đúng
```suggestion
            assert '</p>' in line, line
            new_line = line.replace("<p>","<p align=\"center\">")
```